### PR TITLE
Allow articles, layers and collectory to be refreshed online

### PR DESCRIPTION
**Pull request by charvolant**
Fix for issue https://github.com/AtlasOfLivingAustralia/bie-index/issues/300

Updates can be triggered via existing API

### CODE REVIEW

File: grails-app/services/au/org/ala/bie/ImportService.groovy [hunk 1]
```diff
@@ -567,18 +573,20 @@ class ImportService implements GrailsConfigurationAware {
...
     }
 
     /**
      * Index Knowledge Base pages.
      */
-    def importKnowledgeBasePages() throws Exception {
-        log "Starting knowledge base import"
-        // clear the existing WP index
-        indexService.deleteFromIndex(IndexDocType.KNOWLEDGEBASE)
+    def importKnowledgeBasePages(boolean online) throws Exception {
```
Comment (nickdos) [inline grails-app/services/au/org/ala/bie/ImportService.groovy:588]: Doe it need the `boolean online = false` here like you've got below?

File: src/main/resources/public/openapi.yml [hunk 2]
```diff
@@ -912,9 +920,11 @@ paths:
     get:
       summary: Import workpress pages
       description: >
-        Imports wordpress pages into the offline index
+        Imports wordpress pages into the offline (defauult) or online index
```
Comment (nickdos) [inline src/main/resources/public/openapi.yml:923]: typo `defauult`

**Review by nickdos — CHANGES_REQUESTED** [review]
typo fix

File: src/main/resources/public/openapi.yml (hunk 2 above)
Comment (charvolant) [reply src/main/resources/public/openapi.yml]: Fixed

File: grails-app/services/au/org/ala/bie/ImportService.groovy (hunk 1 above)
Comment (charvolant) [reply grails-app/services/au/org/ala/bie/ImportService.groovy:588]: IndexService parameters have been left without defaults, to allow the controller to explicitly set it.

**Review by nickdos — APPROVED** [review]

**Comment by nickdos** [comment]
Good to merge ✅

**Pull request merged**